src/pages/ExamsPage/data/api.js

```diff
@@ -50,3 +50,10 @@ export async function createAllowance(courseId, data) {
   const response = await getAuthenticatedHttpClient().post(url, data);
   return response.data;
 }
+
+export async function editAllowance(courseId, allowanceId, extraTimeMins) {
```

Comment:
this is just the same code as create, maybe we just call create in the edit hook? Kind of saves us from adding and testing duplicate code.